examples/mutual-ssl/mutual_ssl.bal

```diff
@@ -2,9 +2,8 @@ import ballerina/http;
 import ballerina/log;
 
 // Create a new service endpoint to accept new connections
-//that are secured via mutual SSL.
+// that are secured via mutual SSL.
```

Comment:
This is an outdated description. Shall we rephrase to something like the following:
"Create an HTTP listener configuration which will configure a listener to accept new connections that are secured via mutual SSL."